types/onfleet__node-onfleet/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for @onfleet/node-onfleet 1.0
+// Project: https://github.com/onfleet/node-onfleet
+// Definitions by: Marco Beltempo <https://github.com/marcobeltempo>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.5
+
+import Onfleet = require('./onfleet');
+
```

Comment:
Is there a way I can easily import types/interfaces from the `Resources` directory using module interop?

For example:
`import Onfleet, { CreateTaskProps } from 'onfleet__node-onfleet';`

Reply:
You should be able to do:

without esModuleInterop:
```ts
import Onfleet = require('@onfleet/node-onfleet');
import Task = require('@onfleet/node-onfleet/Resources/Task');
// and access Task.CreateTaskProps
```

with esModuleInterop:
```ts
import Onfleet from '@onfleet/node-onfleet';
import { CreateTaskProps } from '@onfleet/node-onfleet/Resources/Task';
```

You could conceivably also nest namespaces together so that they’re all accessible from the index:

```ts
import Onfleet = require('@onfleet/node-onfleet');
type T = Onfleet.Task.CreateTaskProps;
```

but you’d have to be careful that those namespaces _only_ contain types, because the actual `Task` class isn’t going to exist on `Onfleet.Task` at runtime. Not sure those hoops are worth jumping through.